docs/attributes-registry/agent.md

```diff
@@ -0,0 +1,21 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Agent
+
+## Agent Attributes
+
+<!-- semconv registry.agent(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `agent.distro` | string | Agent distribution. [1] | `github.com/signalfx/splunk-otel-collector`; `github.com/aws-observability/aws-otel-collector` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `agent.id` | string | Unique identifier of agent instance. [2] | `627cc493-f310-47de-96bd-71410b7dec09` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `agent.type` | string | Agent type. [3] | `io.opentelemetry.collector`; `com.dynatrace.one_agent`; `com.newrelic.infra_agent` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `agent.version` | string | The version string of the agent. The format is not defined by these conventions. | `2.0.0`; `a01dbef8a` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Redundant?
```suggestion
| `agent.version` | string | The version string of the agent. | `2.0.0`; `a01dbef8a` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Reply:
It's copied from [service.version](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/resource#service). I agree it does not add value and can be removed.

Reply:
Okay, I guess we can tackle those together